naughty/rhel-9/2835-pmproxy-selinux

```diff
@@ -0,0 +1 @@
+bg.wait_in_text(".page-body", "Data source is working")
```

Comment:
This is too little context. Please add at least the affected test from the traceback.